assets/component-slider.css

```diff
@@ -64,8 +61,8 @@ slider-component {
     scroll-padding-left: 1rem;
     -webkit-overflow-scrolling: touch;
     margin-bottom: 1rem;
-    padding-top: calc(var(--shadow-vertical-offset) * -1 + var(--shadow-blur-radius));
-    padding-bottom: calc(var(--shadow-vertical-offset) + var(--shadow-blur-radius));
+    padding-top: max(0.5rem, calc(var(--shadow-vertical-offset) * -1 + var(--shadow-blur-radius)));
+    padding-bottom: max(0.5rem, calc(var(--shadow-vertical-offset) + var(--shadow-blur-radius)));
```

Comment:
good point, so should be conditional to a class applied based on the card/standard layout then

Reply:
That's definitely valid and I believe Meli mentioned it in the ticket too. If the card discrepancy is also a sev 1 concern that we want done asap I can play with it before merging this. Otherwise, the current fix in this PR is definitely sev 1 and should be merged separately so it doesn't potentially get held up by reviews/discussions for the card aspect.